### DISCUSSION TITLE
几个模型都是开发工具显示正常 手机真机不正常，多个材质多个贴图，像是错乱了

### DISCUSSION THREAD

**Discussion by baron517** [galacean/discussion#36] [Mini Program（小程序）] [answered]
模型下载地址：https://wifi.pro.youzewang.com/model/jia.glb和https://wifi.pro.youzewang.com/model/jian.glb 
调试工具效果 
![lQLPJx26f--oIvTNA9HNBXCwXsJBVHWRXjcDjzo_pkADAA_1392_977](https://user-images.githubusercontent.com/12295860/206684288-e5c6c6cf-ffd0-497f-a31c-887cadea11f6.png)
真机效果
![lQDPJxuh5eHLunTNBQDNAlCwOpld00OYxr4DjzpKjIB6AA_592_1280](https://user-images.githubusercontent.com/12295860/206684389-863d2a8d-2ab8-4a3a-a25b-00fb67bc0fbf.jpg)

**Comment by baron517** [comment]
第一个：https://wifi.pro.youzewang.com/model/jia.glb
第二个：https://wifi.pro.youzewang.com/model/jian.glb

**Comment by daiqiaojun** [comment, marked as answer]
我用你的模型测试了下，在淘宝小程序里安卓和ios都是好的，支付宝小程序里安卓是好的，ios有出现你上述的问题。我在对比threejs案例测试的时候发现，将你的glb格式转化为gltf格式就没问题了，你可以尝试我说的这个方法。至于支付宝小程序ios显示出问题的具体情况还是需要官方技术去核查下，猜测和glb格式的二进制长度有一定关系。
